index.css

```diff
@@ -0,0 +1,114 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap');
+
+body {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
**OPTIONAL** - Though your portfolio looks good 👌, it is very important that it fits well across all mobile screen sizes `<765px`. I suggest that you use [media queries](https://www.google.com/url?sa=t&source=web&rct=j&url=https://www.w3schools.com/css/css3_mediaqueries.asp&ved=2ahUKEwj8rfeh1sH8AhVO4qQKHQBJCsgQFnoECAcQAQ&usg=AOvVaw34me2GWSMjbN7a7mjFFhmF) to make your site responsive, please fix this 👍 👍

Reply:
@mckent05 
Thanks for the clarity